src/providers/SessionContext.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { AppProviderProps } from './AppProvider';
+
+enum SessionStates {
+  DEFAULT = 'DEFAULT',
```

Comment:
what is this one for? it is not the default value of the session state anyways. I don't expect a fully functioning session handler for this PR but wanted to point out that this state might be pointless.

Reply:
I see your point, it might be simpler just to have `LOCKED|UNLOCKED`.

However, I thought that maybe we wanted to load the session from the local storage and until we validate if the session is still valid it isn’t precisely `LOCKED` nor `UNLOCKED`.

Checking again the api docs, I don’t think that will be the case. I can change it to `LOCKED|UNLOCKED`.